include/astshim/PolyMap.h

```diff
@@ -49,7 +49,7 @@ class PolyMap : public Mapping {
 
 public:
     /**
-    Construct a @ref PolyMap with specified forward and inverse transforms.
+    Construct a @ref PolyMap with specified forward and/or inverse transforms.
```

Comment:
The user may specify either or both of the forward and inverse transforms. This wording is intended to make that clear. A plain "and" would suggest that both must be specified. A plain "or" would suggest that both could be omitted.